noxfile.py

```diff
@@ -315,94 +314,3 @@ def prepare_release(session: nox.Session) -> None:
     next_dev_version = release.get_next_development_version(version)
     release.update_version_file(next_dev_version, VERSION_FILE)
     release.commit_file(session, VERSION_FILE, message="Bump for development")
-
-
-@nox.session(name="build-release")
```

Comment:
Can we keep this env so nox is still in the center of the processes and the manual ways remain in the repo? Plus, consider implementing pinning of the build env on this level per my other comments.